phantom_connector.py

```diff
@@ -923,6 +915,26 @@ def _extract_file(self, action_result, file_path, file_name, recursive, containe
 
         return action_result.set_status(phantom.APP_SUCCESS)
 
+    @staticmethod
+    def is_deflation_supported_file(file_path) -> Tuple[str, bool]:
+        """
+        Checks if the file is supported for deflation.
+
+        This method patches invalid behavior of some Operating
+        Systems recognizing MS Office files (eg. xslx) as zip
+        files which lead to an enormous deflation process run
+        hanging the service.
+        """
+        msooxml_magic_file_path = os.path.join(pathlib.Path(__file__).parent.resolve(), "magic", "msooxml")
```

Comment:
You should be able to use `Path` object without this `join` "cast" to `str`

Reply:
It's not a string cast (you probably think of `str.join()`), it's a join for building an OS-aware path with proper separators between dirs.

Reply:
What I meant was `msooxml_magic_file_path = pathlib.Path(__file__).parent.resolve() / "magic" / "msooxml"` Should work as well.

Reply:
Oh, I did not know this notation 😱 Nice one!